lib/ansible/modules/network/nxos/nxos_vlan.py

```diff
@@ -254,10 +254,10 @@ def get_vlan(vlanid, module):
 
 def apply_key_map(key_map, table):
     new_dict = {}
-    for key, value in table.items():
+    for key in table:
         new_key = key_map.get(key)
         if new_key:
-            new_dict[new_key] = str(value)
+            new_dict[new_key] = str(table.get(key))
```

Comment:
Just did that as I thought if we are trying to follow the same in all other place.
Reverted it back to the earlier one.